olcPGEX_AnimatedSprite.h

```diff
@@ -115,7 +117,7 @@ namespace olc
 	{
 		frameTimer += fElapsedTime;
 
-		if (frameTimer >= frameDuration) {
+		if (frameTimer >= frameDurations[state]) {
```

Comment:
I think this should handle states that use the default duration.

Reply:
The default duration will be set into the frameDurations array when the state is added.